modules/packages/Socket.chpl

```diff
@@ -1,6 +1,5 @@
 /*
- * Copyright 2020-2021 Hewlett Packard Enterprise Development LP
- * Copyright 2004-2019 Cray Inc.
+ * Copyright 2021-2022 Hewlett Packard Enterprise Development LP
```

Comment:
Why are the previous copyright years removed?

Reply:
It looked like running the update-copyright script just added the 2022 HPE copyright, so I inferred that the Cray copyright is not necessary for new code. I changed it to 2021-2022 since development on this started in 2021. I am happy to adjust it if I did something wrong though.

Reply:
I'd missed this was a recently added file, so never mind